api/envoy/api/v2/auth/cert.proto

```diff
@@ -119,6 +120,8 @@ message PrivateKeyProvider {
 
 // [#next-free-field: 7]
 message TlsCertificate {
+  option (envoy.protobuf.redacted) = true;
```

Comment:
You should redact only `private_key` and `password`, rest isn't sensitive.

Reply:
Makes sense. I'll change the `redacted` option to apply per-field.